### PR TITLE
Fixes the Lavaland Gym

### PR DESCRIPTION
**Pull request by KingDairy**
He very clearly says "ah, fuck you, leatherman" but I don't want to get into a wrestling match over it so I'll just leave "leatherhead" in.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
DOES THIS EVEN WORK

**Comment by Bawhoppen** [comment]
who knows we'll find out soon

**Comment by KingDairy** [comment]
http://i.imgur.com/0QZcl0z.png
FUCK
TIME TO FIGURE OUT HOW TO DO THIS RIGHT

**Comment by KingDairy** [comment]
Okay, so pick() does not work like that. This should work, and by that I mean I tested it and it looks fine.

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 1]
```diff
@@ -33,7 +33,12 @@
 	layer = MOB_LAYER - 0.1
 	var/removingairlock = 0
 
-
+/mob/living/simple_animal/hostile/zombie/darkholme //BOY♂NEXT♂RUIN
```
Comment (optimumtact) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: meme comment detected

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: Could you please add goofs new macho man wrestling belt to the mobs butcher results?

/obj/item/weapon/storage/belt/champion/wrestling

That or just throw one or two on the floor in the gym

Comment (KazeEspada) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: It should be the boss of this gym, not the leatherman who has the belt

File: _maps/RandomRuins/LavaRuins/lavaland_gym.dmm [hunk 2]
```diff
@@ -15,7 +15,7 @@
 "o" = (/obj/structure/stacklifter,/turf/open/floor/plasteel/freezer{baseturf = /turf/open/floor/plating/lava/smooth},/area/ruin/powered)
 "p" = (/obj/structure/window,/obj/structure/window{icon_state = "window";dir = 8},/obj/effect/decal/cleanable/blood,/turf/open/floor/plasteel/freezer{baseturf = /turf/open/floor/plating/lava/smooth},/area/ruin/powered)
 "q" = (/obj/structure/window,/turf/open/floor/plasteel/freezer{baseturf = /turf/open/floor/plating/lava/smooth},/area/ruin/powered)
-"r" = (/obj/structure/window,/obj/item/weapon/shard,/mob/living/simple_animal/hostile/zombie{name = pick("Leatherman","Leatherhead")},/turf/open/floor/plasteel/freezer{baseturf = /turf/open/floor/plating/lava/smooth},/area/ruin/powered)
```
Comment (xxalpha) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: >programming in the map file

you mad man

Comment (RemieRichards) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: A DMM file is just a normal DM file, I am however unsure if it actually runs any DM code it finds in there.

Comment (KingDairy) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: I'm sure if I actually had some idea of what I was doing I would have known that wouldn't work.

Comment (xxalpha) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: No it's not Remie, DM is very specific with its file formats.

Comment (RemieRichards) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: @xxalpha Yes it is, it's a normal DM file, there's literally nothing special about it beyond the structure of the text inside it, please don't try to correct people when you're wrong.

Comment (xxalpha) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: Go ahead and try to open a dmm file if it is not called dmm.

Comment (RemieRichards) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: That doesn't stop it being a normal DM file, you complete tool.

Comment (optimumtact) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: no need to continue this argument any further guys

Comment (xxalpha) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: Now you're just being toxic remie. I doubt DM compiles a map file with code in it.

Comment (RemieRichards) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: Omg, learn to read
>I am however unsure if it actually runs any DM code it finds in there.

Comment (KingDairy) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: Given that my attempt at randomizing the name didn't work, I'm assuming it doesn't.

**Comment by Shapsy** [comment]
holy shit it actually is leatherman

**Comment by Iamgoofball** [comment]
fuck you its LEATHERHEAD

**Comment by Chocobro** [comment]
https://youtu.be/uWMuOrfBgDE?t=20s
it's leatherhead

**Comment by KingDairy** [comment]
No, it's definitely leatherman.
I might go back and fix the grammar on this, leather___ shouldn't be capitalized and "boss of this gym" should probably be "the boss of this gym"

**Pull request merged**